### DISCUSSION TITLE
Welcome to herdr Discussions!

### DISCUSSION THREAD

**Discussion by nackko** [f8full/herdr#32] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nackko** [comment]
👋 @vince-fabmob ;)

**Comment by vince-fabmob** [comment]
Here's some links to a Herdr data conversion into GeoJson file. 

https://github.com/savoirfairelinux/fabmob/commit/4b8ef0519c8d4aed25487e41f919bf79aef3b55a

this page explain how we can use Herdr to track trips and visualize it.
https://wiki.lafabriquedesmobilites.fr/wiki/Cr%C3%A9ation_d%27outils_de_conversion_de_traces_de_trajets_GPS
